src/_includes/common/filter-events.md

```diff
@@ -0,0 +1,130 @@
+
+### Proper Event Grouping
+
+Proper event grouping maintains a meaningful issue stream and reduces redundant notifications. Sentry groups similar *events* into unique *issues* based on their *fingerprint*. An event’s fingerprint relies firstly on its **stack trace**.
+
+{{ include.filter-stack-trace }}
+<!-- Guideline: as appropriate, add the example and screen shot for the SDK your are documenting -->
+
+### Apply Workflows
+
+Event streams with inbound filter and proper event grouping can be made more efficient by practicing good development hygiene. When Sentry alerts you to an issue in your code, resolve it or discard. 
+
+### Rate Limit
+
+Rate limits allow you to set the maximum volume of events a project will accept during a period of time. This type of limit can be useful for managing your monthly event quota, but once a defined threshold is crossed, **subsequent events will be dropped**. You shouldn't constantly hit your rate limit. Instead, regard it as a ceiling intended to protect you from unexpected spikes.
+
+Under `[Project Settings] » Client Keys » Configure`, you can create multiple DSN keys per-project and assign different (or no) limits to each key. This will allow you to dynamically allocate keys (with varying thresholds) depending on Release, Environment, and so forth.
+
+For example, you may have a project in production that generates many alerts. A rate limit allows you to set the maximum amount of data to 500 events per minute, for instance. Additionally, you can create a second key for the same project for your staging environment, which is unlimited, ensuring your QA process is still untouched.
+
+## Configure your SDK to Filter Events
+
+Configure your SDK to filter events by using the `beforeSend` callback method and configuring, enabling, or disabling integrations.
+
+### Using `before-send`
+
+All Sentry SDKs support the `beforeSend` callback method. `before-send` is called immediately before the event is sent to the server, so it’s the final place where you can edit its data. It receives the event object as a parameter, so you can use that to modify the event’s data or drop it completely (by returning `null`) based on custom logic and the data available on the event.
+
+{{ include.filter-init_content }}
+<!-- Guideline: add the `init` call for the SDK your are documenting -->
+
+The `before-send` callback is passed both the `event` and a second argument, `hint`, that holds one or more hints. 
+
+Typically a `hint` holds the original exception so that additional data can be extracted or grouping is affected. In this example, the fingerprint is forced to a common value if an exception of a certain type has been caught:
+
+{{ include.filter-example_content }}
+<!-- Guideline: add a code sample that supports this example for the SDK your are documenting -->
+
+
+Note also that breadcrumbs can be filtered, as discussed in Understand Breadcrumbs <!-- TO DO : add link-->
```

Comment:
```suggestion
Note also that breadcrumbs can be filtered, as discussed in Understand Breadcrumbs <!-- TODO : add link-->
```

(all programming languages use `TODO` tag instead of `TO DO`, this way most tools can find them automatically)

Reply:
Ah, this is an internal note to me to fix this later. But I should wrap it in the {% comment %} syntax. Thanks.